release_notes.txt

```diff
@@ -1,4 +1,5 @@
 ### Version 0.0.19 ###
 
-Added a permute vertex
-Added the release notes text file to the repo
\ No newline at end of file
+* Added a permute vertex
+* Added the release notes text file to the repo
+* Added __version__ attribute to Python
```

Comment:
I think the newest change goes to top (assuming adding the release notes text file was the first addition). Small nit though.

Reply:
Good point, I will swap it around